themes/openy_themes/openy_rose/templates/page/page.html.twig

```diff
@@ -152,7 +152,10 @@
     <footer class="footer">
       <div class="container">
         <div class="col-sm-2 col-xs-4 hidden-sm hidden-md">
-          <a href="{{ base }}" class="footer__brand brand" title="{{ "YMCA"|t }}"></a>
+          {% if footer_logo %}
+            {% set background_image = "style='background-image: url(" ~ file_url(footer_logo) ~ ")'" %}
+          {% endif %}
+          <a href="{{ base }}" class="footer__brand brand" title="{{ "YMCA"|t }}" {{ background_image|raw }}></a>
```

Comment:
If variable `footer_logo` is undefined then `background_image ` is undefined as well. You need to check this before using variable.